server/@types/forms/index.d.ts

```diff
@@ -1,6 +1,5 @@
 declare module 'forms' {
   export interface CreateGoalForm {
-    readonly prisonNumber: string
```

Comment:
Was this causing you some grief somewhere?

TBH I was kinda thinking we might use that in some near-future dev for this story .... IE. I'm thinking we need to write something that says "when you arrive on the AddStep page (and the AddNote page), check you have a CreateGoalForm in the session AND its for the same prison number as in the URL address)

My thinking is we need to make sure you dont just land on the AddStep or AddNote page without having gone through the previous pages (ie. people bookmarking / typing urls etc); and also we should check that if you've started the journey for prisoner X you complete it for prisoner X

Reply:
It wasn't causing a problem, just some confusion as it was `prisonNumber` was effectively duplicated in the controller. I can revert it if you like, or we can add it when we need it.

Reply:
If you could revert it that would be good. I think I might start on the code that will use it today ...